### DISCUSSION TITLE
DataTable show column value as time duration(0 days 00:00:00.000000068)

### DISCUSSION THREAD

**Discussion by blnprasad** [datapane/datapane#77] [General]
![sample](https://user-images.githubusercontent.com/9347569/107004166-e7c22780-67b3-11eb-99f4-ae93908cc463.png)

![image (1)](https://user-images.githubusercontent.com/9347569/107004040-af224e00-67b3-11eb-9500-928d3c4ac318.png)

Im converting below dictionary( json array) to DF and trying to generate table and it seems that one particular column being showed as time duration in report.
 data = [{'a': 0, 'b': 8.222222222222221, 'c': -107.0, 'd': "68"}]
 df = pd.DataFrame.from_dict(ch_table)

in the report i see 'd' column being displayed as  time duration(0 days 00:00:00.000000068).

can you please help me to find whats going wrong here.

**Comment by mands** [comment]
Can you confirm your version of datapane and try upgrading it (`pip install --upgrade datapane`) as many time/date related issues were fixed in version 0.9.2

**Reply by blnprasad** [reply]
I was using 0.9.0. with 0.9.2 I don't see this issue. Thanks.
